src/builders.ts

```diff
@@ -122,20 +122,31 @@ export async function broadcastTransaction(
   transaction: StacksTransaction,
   network: StacksNetwork
 ): Promise<string> {
-  const tx = transaction.serialize();
+  const rawTx = transaction.serialize();
+  const url = network.getBroadcastApiUrl();
+
+  return broadcastRawTransaction(rawTx, url);
+}
```

Comment:
I'm curious if it makes sense to pass in just the url like this. It'd be nice if both functions were the same, either using the url as a param, or both taking network as a param. Open to either or keeping it like this. What are your thoughts @yknl ?

Reply:
I think having it take a URL makes it more flexible. In case you don't want to or cannot create a network config for where you want to broadcast this transaction.

Reply:
That makes sense to me! Does that mean `broadcastTransaction` would also only take the url, as that is the only thing its using from `network`?

Reply:
I expect that people using `broadcastTransaction` should already have a network config instantiated with their transaction so it would make more sense for the function to take a network as arg. The `broadcastRawTransaction` function is more like advanced usage.

Reply:
ok! sounds good to me :)